meinberlin/assets/scss/_variables.scss

```diff
@@ -55,6 +55,7 @@ $font-size-sm: 0.9rem !default;
 $font-size-xs: 0.8rem !default;
 
 // use for min-width media-queries
+$breakpoint-xxs: 23.75rem !default;
```

Comment:
why do you want to add an additional breakpoint? also this is added as a min width value but you use it in a max width query, we don't really want to add additional break points as it add complexity, applying the same style up to a width of 512px is pretty standard, bootstraps first breakpoint is 576 so we aren't far off the norm.